data-model/developer-schema/annotation/0.4.0/schema/annotation-batch-metadata.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://schemas.dissco.tech/schemas/developer-schema/annotation/0.4.0/annotation-batch-metadata.json",
+  "properties": {
+    "ods:placeInBatch": {
+      "description": "For batching only. Links batch metadata to specific annotations in an event. Value must correspond to an annotation's ods:placeInBatch",
+      "type": "integer"
+    },
+    "searchParams": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
```

Comment:
add `additionalProperties:false` in this declaration

Reply:
:+1: